front-end/src/app/users/user-list/user-list.component.html

```diff
@@ -0,0 +1,61 @@
+<p-toast></p-toast>
+
+<div class="card">
+  <p-toolbar styleClass="mb-4">
+    <ng-template pTemplate="left">
+      <h1>Manage Users</h1>
+    </ng-template>
+
+    <ng-template pTemplate="right">
+      <button pButton pRipple label="Add User" icon="pi pi-plus" class="p-button"></button>
+    </ng-template>
+  </p-toolbar>
+
+  <p-table
+    #dt
+    [value]="items"
+    [lazy]="true"
+    (onLazyLoad)="loadTableItems($event)"
+    responsiveLayout="scroll"
+    dataKey="id"
+    [selection]="selectedItems"
+    (selectionChange)="onSelectionChange($event)"
+    [selectAll]="selectAll"
+    (selectAllChange)="onSelectAllChange($event)"
+    [paginator]="true"
+    [rows]="10"
+    [totalRecords]="totalItems"
+    [loading]="loading"
+    [globalFilterFields]="['first_name', 'last_name', 'email', 'role', 'is_active']"
+    currentPageReportTemplate="Showing {first} to {last} of {totalRecords} users"
+    [showCurrentPageReport]="true"
+  >
+    <ng-template pTemplate="caption">
+      <div class="flex align-items-center justify-content-between">
+        <h3 class="m-0">Existing Users</h3>
+        <span class="p-input-icon-left">
+          <em class="pi pi-search"></em>
+          <input pInputText type="text" placeholder="Search..." />
+        </span>
+      </div>
+    </ng-template>
+    <ng-template pTemplate="header">
+      <tr role="row">
+        <th pSortableColumn="name" id="name-column" role="columnheader">Name <p-sortIcon field="name"></p-sortIcon></th>
+        <th pSortableColumn="email" id="email-column" role="columnheader">Email <p-sortIcon field="email"></p-sortIcon></th>
+        <th pSortableColumn="role" id="role-column" role="columnheader">Role <p-sortIcon field="role"></p-sortIcon></th>
+        <th pSortableColumn="is_active" id="active-column" role="columnheader">Status <p-sortIcon field="is_active"></p-sortIcon></th>
+      </tr>
+    </ng-template>
+    <ng-template pTemplate="body" let-item>
+      <tr role="row">
+        <td>{{ item.last_name }}, {{ item.first_name }}</td>
```

Comment:
I noticed that you added a `name` alias to the django view, which combines the first and last names into a third field.  We should stick to one way of combining the name fields.  I'm on the fence over whether to do it on the front end or on the backend.  Which do you think is better?

Reply:
In an outside conversation, we've agreed that this is not an issue as the `name` annotation from the back-end is only used for sorting and does not get passed by the serializer to the front-end.